sortinghat/core/errors.py

```diff
@@ -79,3 +80,10 @@ class ClosedTransactionError(BaseError):
 
     code = CODE_CLOSED_TRANSACTION_ERROR
     message = "%(msg)s"
+
+
+class LockedIdentityError(BaseError):
+    """Exception raised when performing a change on a locked unique identity"""
+
+    code = CODE_LOCKED_IDENTITY_ERROR
+    message = "%(msg)s"
```

Comment:
The exception should include the **uuid** of the identity locked.

Reply:
Done.